theme/styles/_variables.sass

```diff
@@ -33,17 +36,21 @@ $c-text-light: #676767
 $c-text-back: #FFF
 
 // Breakpoints
-$break320: '(min-width: 320px)'
-$break480: '(min-width: 480px)'
-$break768: '(min-width: 768px)'
-$break992: '(min-width: 992px)'
-$break1200: '(min-width: 1200px)'
-
-$break320M: '(max-width: 319px)'
-$break480M: '(max-width: 479px)'
-$break768M: '(max-width: 767px)'
-$break992M: '(max-width: 991px)'
-$break1200M: '(max-width: 1199px)'
+$break320: 20em
+$break480: 30em
+$break768: 48em
+$break992: 62em
+$break1200: 75em
+$break1296: 81em
+$break1400: 87em
+
+$break320M: emToMem(20em - $w-em-epsilon)
+$break480M: emToMem(30em - $w-em-epsilon)
+$break768M: emToMem(48em - $w-em-epsilon)
+$break992M: emToMem(62em - $w-em-epsilon)
+$break1200M: emToMem(75em - $w-em-epsilon)
+$break1296M: emToMem(81em - $w-em-epsilon)
+$break1400M: emToMem(87em - $w-em-epsilon)
```

Comment:
`Mem` jednotku nechápu. Použití tady ano, ale je vidět, že tak se nedá tak elegantně odečíst jeden pixel.

Reply:
Možná by `M` mohlo automaticky odečítat `0.1px`?

Reply:
a `$w-em-epsilon` by mohlo být spíš `em(.1px)` to nám vydrží, než bude 10x pixel density 🤡